.eslint-doc-generatorrc.mjs

```diff
@@ -0,0 +1,17 @@
+import prettier from "prettier";
+
+/** @type {import('eslint-doc-generator').GenerateOptions} */
+const config = {
+	configEmoji: [
+		["recommended", "✅"],
+		["legacy-recommended", "✔️"],
+	],
```

Comment:
These will need to be added as an Addon to create-typescript-app's `blockESLintPlugin`. Re-running `npx create-typescript-app` clears them.

Reply:
https://github.com/JoshuaKGoldberg/create-typescript-app/issues/2190